hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -633,6 +656,9 @@ def create_volume(self, volume):
             extras = {'comment': json.dumps(comments),
                       'tpvv': tpvv, }
 
+            if len(self.config.hpe3par_snapcpg):
+                extras['snapCPG'] = self.config.hpe3par_snapcpg[0]
```

Comment:
- if hpe3par_snapcpg is not given in /etc/hpedockerplugin/hpe.conf, will that be defaulted to the value in hpe3par_cpg